test/adapters/IDB_test.js

```diff
@@ -411,8 +411,26 @@ describe("adapter.IDB", () => {
     });
   });
 
-  /** @test {IDB#loadDump} */
-  describe("#loadDump", () => {
+  /**
+   * @deprecated
+   * @test {IDB#loadDump}
+   */
+  describe("Deprecated #loadDump", () => {
+    it("should call importBulk", () => {
+      sandbox.stub(db, "prepare").callsFake(async (name, callback, options) => {
+        return callback({
+          put() {
+            throw new Error("transaction error");
+          },
+        });
+      });
+      return db.loadDump([{ foo: "bar" }]).should.be.a("promise");
+      //return ib.should.have.been.calledOnce;
```

Comment:
This test just checks that a promise was returned. I think `ib.should.have.been.calledOnce` seems more like a test that it called `importBulk`. There are a bunch of tests in `tests/collection_test.js` that do what I think you should do here -- just grep for `sinon.assert.calledOnce` or `sinon.assert.calledWithExactly`. I think you can stub `db.importBulk` instead of `db.prepare`. The return value of `stub()` is the stub/spy that you can use with `calledOnce`.

Reply:
I have changed both tests to stub and confirm that `importBulk` is being called.